sonic_platform_base/module_base.py

```diff
@@ -52,6 +52,8 @@ class ModuleBase(device_base.DeviceBase):
     MODULE_REBOOT_FPGA_COMPLEX = "FPGA"
     # Module reboot type to reboot DPU
     MODULE_REBOOT_DPU = "DPU"
+    # Module reboot type to reboot SMART SWITCH
```

Comment:
@vvolam the module class is for pluggable or non-pluggable modules. We already have DPU reboot. Why we need MODULE_REBOOT_SMARTSWITCH?

Reply:
@prgeor As updated in the https://github.com/sonic-net/SONiC/pull/1847, MODULE_REBOOT_SMARTSWITCH is the reboot type when rebooting the DPU and when entire switch is going for a reboot.